tensorflow/python/compiler/tensorrt/test/quantization_mnist_test.py

```diff
@@ -39,14 +42,42 @@
 from tensorflow.python.ops.losses import losses
 from tensorflow.python.platform import test
 from tensorflow.python.platform import tf_logging as logging
+from tensorflow.python.saved_model import builder
+from tensorflow.python.saved_model import signature_constants
+from tensorflow.python.saved_model import tag_constants
+from tensorflow.python.saved_model import utils as saved_model_utils
+from tensorflow.python.saved_model import signature_def_utils
+from tensorflow.python.saved_model.load import load as saved_model_load
 from tensorflow.python.summary import summary
 from tensorflow.python.training import saver
 from tensorflow.python.training.adam import AdamOptimizer
 from tensorflow.python.training.checkpoint_management import latest_checkpoint
 from tensorflow.python.training.training_util import get_global_step
+from tensorflow.compat.v1 import Session
```

Comment:
This line doesn't work, you need to replace it with 
from tensorflow.python.client import session

then replace "Session(..)" with "session.Session(...)"

see here for more details
https://github.com/tensorflow/tensorflow/blob/8b202f08d52e8206af2bdb2112a62fafbc546ec7/tensorflow/python/client/session_test.py#L31

Reply:
Fixed.